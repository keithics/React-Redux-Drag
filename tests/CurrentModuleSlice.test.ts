import {
  CurrentModuleReducer,
  moduleInitialState,
  setCurrentModule,
} from '../src/redux/slices/CurrentModuleSlice';

describe('Current Module Tests', () => {
  test('should return the initial state', () => {
    // @ts-ignore
    expect(CurrentModuleReducer(undefined, {})).toMatchSnapshot();
  });

  test('should be able to change module', () => {
    expect(CurrentModuleReducer(moduleInitialState, setCurrentModule({ id: 2 }))).toMatchSnapshot();
  });
});
