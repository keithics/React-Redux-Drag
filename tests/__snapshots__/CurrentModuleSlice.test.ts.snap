// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current Module Tests should be able to change module 1`] = `
Object {
  "id": 2,
}
`;

exports[`Current Module Tests should return the initial state 1`] = `
Object {
  "id": 0,
}
`;
